lib/spack/spack/cmd/python.py

```diff
@@ -33,6 +33,9 @@ def setup_parser(subparser):
     subparser.add_argument(
         '-m', dest='module', action='store',
         help='run library module as a script')
+    subparser.add_argument(
+        '--path', action='store_true', dest='show_path',
```

Comment:
Should we do `-p` for a short option?

Reply:
trying not to interfere with actual python interpreter options (`-p` is not one of them, but they're all single-char things, and one that we want to pass through might emerge).